src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('li')];
```

Comment:
There is a potential issue with your code. You are getting all 'li' elements from the document. This could potentially select unwanted elements if there are other 'li' elements elsewhere in your document. It would be better to be more specific with your selector, for example by using a class or id to select the specific list you want to sort.